tests/integration-tests/reports/reportsApi-test.js

```diff
@@ -624,6 +624,69 @@ describe('Integration tests for the reports api', function () {
                 should(headers).eql(EXPORTED_REPORT_HEADER);
             });
 
+            it('Post full cycle stats, export report and check report contents', async function () {
+                const phaseStartedStatsResponse = await reportsRequestCreator.postStats(testId, reportId, statsGenerator.generateStats('started_phase', runnerId));
+                should(phaseStartedStatsResponse.statusCode).be.eql(204);
+
+                const getReport = await reportsRequestCreator.getReport(testId, reportId);
+                should(getReport.statusCode).be.eql(200);
+                const testStartTime = new Date(getReport.body.start_time);
+
+                const statDateFirst = new Date(testStartTime).setSeconds(testStartTime.getSeconds() + 20);
```

Comment:
maybe better it with forEach or for even
as this is the same code with different seconds/startDate

Reply:
Thank you for the suggestion. This change has been done.